templates/visuals/slicer/src/visual.ts

```diff
@@ -25,33 +25,36 @@
  */
 
 module powerbi.extensibility.visual {
+    "use strict";
     export class Visual implements IVisual {
         private target: HTMLElement;
         private updateCount: number;
         private settings: VisualSettings;
 
         constructor(options: VisualConstructorOptions) {
-            console.log('Visual constructor', options);
+            console.log("Visual constructor", options);
             this.target = options.element;
             this.updateCount = 0;
         }
 
         public update(options: VisualUpdateOptions) {
-            console.log('Visual update', options);
+            if (!options || !options.dataViews) {
```

Comment:
DataViewParser is able to handle null and undefined values. However, it's good to include this condition anyway